src/System.Security.Cryptography.X509Certificates/tests/TestEnvironmentConfiguration.Unix.cs

```diff
@@ -0,0 +1,85 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    internal static partial class TestEnvironmentConfiguration
+    {
+        static partial void DetermineCanModifyStores(ref bool canModify)
```

Comment:
Yep, a partial method has to be able to not run and the program make sense.  So no returns, no `out`, but `ref` is okay :).  The other function was because I wanted the more succinct `return false;` than `canModify = false; return;`